src/components/ContactsList.js

```diff
@@ -1,29 +1,68 @@
-import { useState } from "react";
+import { useState, Fragment } from "react";
 import { Link, useSearchParams } from "react-router-dom";
+import * as Loader from "react-loader-spinner";
 
 function ContactsList(props) {
-  //"contacts" must be passed as prop to this component
-  const { contacts } = props;
+  const { contacts, setContacts, loading } = props;
+
+  const handleDelete = (contact) => {
+    console.log("delete me");
+    if (contact.id) {
+      fetch(`http://localhost:4000/contacts/${contact.id}`, {
+        method: "DELETE",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(contacts),
+      })
+        .then((res) => res.json())
+        .then((json) => {
+          const updatedContacts = contacts.filter(
+            (record) => record.id !== contact.id
+          );
+          setContacts(updatedContacts);
+        });
```

Comment:
like how you filtered out the deleted data

Reply:
I don't know why but I read:
'like how you flirted out the deleted data' 
instead of
'like how you filtered out the deleted data' 🤦🏽‍♀️